Adds a new package: cherrytree - A hierarchical note taking application

arno01 opened the pull request:
cherrytree - A hierarchical note taking application, featuring rich text
and syntax highlighting, storing data in a single xml or sqlite file.

This commit also adds PyGtkSourceView - a Python wrapper for the GtkSourceView widget library.

Official page http://www.giuspen.com/cherrytree

pSub commented on pkgs/applications/misc/cherrytree/default.nix:
```diff
@@ -0,0 +1,36 @@
+  doCheck = false;
+
+  meta = {
+    description = "A hierarchical note taking application, featuring rich text and syntax highlighting, storing data in a single xml or sqlite file.";
```
`description` should not end with a dot. See http://nixos.org/nixpkgs/manual/#idm140737319319584.

Commit:
Adds a new package: cherrytree - A hierarchical note taking application

cherrytree - A hierarchical note taking application, featuring rich text
and syntax highlighting, storing data in a single xml or sqlite file.

This commit also adds PyGtkSourceView - a Python wrapper for the GtkSourceView widget library.

Official page http://www.giuspen.com/cherrytree

arno01 commented:
@pSub I've corrected the description (removed period)

lethalman commented:
Travis passed, thanks.
